### ISSUE TITLE
Training code

### ISSUE THREAD

**Issue by FBehrad** [rw-ocrl/ftdinosaur-inference#3]
Thank you so much for the great paper and for sharing your weights.

Can you please also share your training code and metric implementation? 

I followed all the steps you mentioned in the paper, but unfortunately, my performance is poorer than yours. I don't understand what is wrong, as I follow everything. 

Thank you so much.
